### ISSUE TITLE
Steam Cloud: Unable to Sync after Uninstall

### ISSUE THREAD

**Issue by pparmy** [CryoByte33/steam-deck-utilities#67]
I had issues with steamOS reinstalling shaders for some games upon reboot for my Windows/steamOS Deck. I only noticed that after updating the utilities and uninstalled it today in an attempt to see if it would solve the issue. Instead I receive a "STEAM CLOUD: UNABLE TO SYNC" banner below the green play button after selecting a game. In specific Hollow Knight, God of War, Don't Starve Together, and Fallout: New Vegas all have this issue.

When selecting it and choosing retry it starts to "Download..." then reverts back to the error. Disabling cloud saves for these games doesn't work either as it only starts to attempt to open the game and promptly crashes. Switching to desktop mode and hard booting didn't seem to sway anything in favor either.

**Comment by pparmy** [comment]
Verifying File Integrity also hasn't helped nor has reinstalling Cryo Utilities and running shadercashe sync

**Comment by CryoByte33** [comment]
This sounds like the shaders were synced, and then the game was deleted or something, I think the problem you're seeing is that there's a dangling symlink. 

You should be able to fix it by using the "cleanup" function of CU and selecting the affected game(s) to delete shaders, prefixes and symlinks.

**Comment by pparmy** [comment]
I forgot to note that none of the games affected by this are shown when selecting the "Storage" tab and selecting "Clean" under "Delete Game Data"

I have tried to delete the game via Windows but have had no success (though from the way you describe it I'm not even sure if that would help).

**Comment by CryoByte33** [comment]
That's interesting, are they Steam games, or are you adding them as non-Steam games?

**Comment by lppinto** [comment]
Just fixed it by running CU -> Storage and clean up for those two games (which actually appear on the list)

**Comment by CryoByte33** [comment]
Ah, so it seems like there was a dangling symlink left in the steam directories.

Did you hotswap your SD card after doing the data sync?

**Comment by lppinto** [comment]
No, just ran your tools, and then those too games started complaining. Rebooted, same issue. Even reinstalled one of them to see if it  got fixed (on a different drive),  same issue. This only went away when I re-ran your tools and cleaned those to games! Now it's fixed!

**Comment by CryoByte33** [comment]
Very interesting! Something clearly messed up during the first sync but I'm not sure what it was. Glad that it's fixed, though 😅
